rwslib/builders/metadata.py

```diff
@@ -506,10 +569,14 @@ def build(self, builder):
 
     def __lshift__(self, other):
         """Override << operator"""
-        if not isinstance(other, (FormRef, Alias,)):
-            raise ValueError('StudyEventDef cannot accept a {0} as a child element'.format(other.__class__.__name__))
-        self.set_list_attribute(other, FormRef, 'formrefs')
-        self.set_list_attribute(other, Alias, 'aliases')
+        if not isinstance(other, (FormRef, Alias)):
```

Comment:
I really wish I'd made this a decorator. It would clear up a lot of boilerplate I think.